doc/users/interactive.rst

```diff
@@ -146,45 +139,43 @@ Interactive mode controls:
 
-- newly created figures will be shown immediately
-- figures will automatically redraw on change
-- pyplot.show will return immediately by default
+- newly created figures will be displayed immediately
+- figures will automatically redraw when elements are changed
+- `pyplot.show` displays the figures and immediately returns to the prompt
 
-If not in interactive mode then:
+Not in interactive mode:
 
-- newly created figures and changes to figures will
-  not be reflected until explicitly asked to be
-- pyplot.show runs the GUI event loop and does not return until all of
-  the plot windows are closed
+- newly created figures and changes to figures are not displayed until 
+`.pyplot.show` is called again or `.pyplot.pause` exits
+- `pyplot.show` runs the GUI event loop and does not return until all 
+the plot windows are closed
 
 
 If you are in non-interactive mode (or created figures while in
 non-interactive mode) you may need to explicitly call `.pyplot.show`
-to bring the windows onto your screen.  If you only want to run the
-GUI event loop for a fixed amount of time you can use `.pyplot.pause`.
-This will both block the progress of your code (as if you had called
-`time.sleep`), ensure the current window is shown and if needed
-re-drawn, and run the GUI event loop (so the windows are "live" for
-interaction) for the specified period of time.
-
-Being in "interactive mode" is orthogonal to the GUI event loop being
-integrated with your command prompt.  If you use `pyplot.ion`, but
-have not arranged for the event loop integration, your figures will
-appear but will not be "live" while the prompt is waiting for input.
+to display the windows on your screen.  If you only want to run the
+GUI event loop for a fixed amount of time, you can use `.pyplot.pause`.
+This will block the progress of your code as if you had called
+`time.sleep`, ensure the current window is shown and re-drawn if needed, 
+and run the GUI event loop (so the windows are "live" for
+interactive modification) for the specified period of time.
+
+The GUI event loop being integrated with your command prompt and 
+the figures being in interactive mode are independent of each other. 
+If you use `pyplot.ion` but have not arranged for the event loop integration, 
+your figures will appear but will not be "live" while the prompt is waiting for input. 
 You will not be able to pan/zoom and the figure may not even render
 (the window might appear black, transparent, or as a snapshot of the
 desktop under it).  Conversely, if you configure the event loop
 integration, displayed figures will be "live" while waiting for input
-at the prompt, regardless of pyplot's "interactive mode".  In either
-case, the figures will be "live" if you use
-``pyplot.show(block=True)``, `.pyplot.pause`, or run the the GUI main
-loop in some other way.
+at the prompt, regardless of pyplot's "interactive mode". 
+
+The figures will also be "live" if you use ``pyplot.show(block=True)``, 
```

Comment:
```suggestion
It any case the figures will be "live" if you use ``pyplot.show(block=True)``, 
```

Reply:
*In* any case,

Reply:
in any case is an ambigious reference 'cause it's not clear what the cases are ...